set.mm

```diff
@@ -531785,7 +531916,182 @@ can be drawn starting from two points (we take ` 0 ` and ` 1 ` ), and taking
+        cmul dvdsprmpweq imp simprd constrextdg2 cconstr isconstr sylib ) ADUIU
+        LZEUMZUNZMFUOUPZUQJULURUPUSJUTVAZUIVBAUWLVBUNZVFZUWNVFZVCUJULZUMZVGUSZU
+        WMUWTVDUMZVEZVFZUWPUJVHVIUMZGVJZUWSUWTUXGUNZVFZUXEVFZVHUXCVKUPZVHVGVKUP
+        ZUOUPZUQUKULZURUPZUSZUWPUKUTUXJUXNUTUNZVFZUXPVFZUQVLUNZUWOVTUNZUXQUWOUX
+        OVMVNZUWPUXTUXSVOVPUXSUWOVHVHVGDVQZVRZVSUPZVKUPZUXLUOUPZVTMUYFFUXLUOUGU
+        FWAZUXSUYGUXSUYFUXLWBVNZUYGWCUNZUXJUYIUXQUXPUXJUYCWDVHVGUXLUYFWEUXLWFZU
+        YFWFVHWGUNUXJWHVPZVGUXFUNZUXJUYMVHWKUNZVGVHWIUMUNZUXLWKUNZWJUYOUYPWLWMU
+        YOUYPWLWNVHVGWOWPVPUWSUYCWDVEUXHUXEUWSDWDUWRUWMWDDUWRBCEHIUWLOPQRSTUAUW
+        QUWLWQUNAUWLXAWRWSWTXBZXCXDZXCZUYFUXLXEZXFUXSUXMUYGUXKUYFUOUPZUXSUXMUXS
+        UXMUXOXGUXRUXPXHZUXSUQUXNUQXGUNUXSXIVPUXJUXQUXPXJZXKZXLXMZUXSUYIUYJUYGX
+        NUNZUYSUYTUYGXOXPZUXSVUAWCUNZVUAXNUNZUXSUXKUYFWBVNZVUHUXJVUJUXQUXPUXJUX
+        KUXKUYEVKUPZUYFWBUXJUXKYDUMZUXKUYEVULWFZUXJUXKUXLWBVNZUXKWGUNUXJVUNUXPU
+        KUTVAZUXJUXLGUWTHIUKKLUXKVHUBUCUDUWSUXHUXEXJZUYLUXJUXAVGVHVKUXIUXBUXDXQ
+        ZXRUXJUXKXSUXJUWTUXGVUPUXJUWTXTUXJUWTXTUSZUXAXTUSUXJVURVFZUXAVCXTUMZXTV
+        USVCUWTXTUXJVURXHYAVUTXTUSVUSVCYEVPYBVUSUXAXTVUSUXAVGXTUXJUXBVURVUQYCVG
+        XTYFVUSYGVGYGVTUNYGVGUNYHYGUUAUUBUUCVPUUDUUEUUFUUHZUUGUUIZUUJZUXKUXLUUK
+        XFUXJUXCUXFUNZUYEUXFUNZUYEUXCVEZUYEUXKVIUMZUNZUXJUWTUXFUULUNUWTXTYFVVDU
+        XJUXFUWTGVUPUUMVVAUXFUWTUUNYIZUXJWDUYDVHWEUYNUXJWJVPZUWSUYDWDVEUXHUXEUW
+        SVGUYCWDVGWDVEUWSUUOVPUYQYJXCUUPUXJWDUXCUYDVHWEVVJVVIUXJVGUYCUXCUXJVGVU
+        LUXCUXJVGVVGUNVGVULVEUXJVGUXKUXLUXLUYKUUQVVCUURVULUXKVGVUMYKXFUXJUXCWDV
+        EZUXCVULUSUXJVVDVVKVVIWDVHUXCWEYKXFUXCWDUXKVHUXKWFZWEUUSXFUUTUXJDUXCUXJ
+        UWMUXCDUXIUXBUXDUVAUWRUWNUXHUXEUVBUVCXBYJUVDZVVDVVHVVEVVFVFVVDUXCUYEVHU
+        XKVVLVVDUVEUVFUVGUVHUVOUXJUXCYLUNVVFVUKUYFUSUXJUXCUXFVVIUVIVVMUXCUYEVHY
+        LUVJYIUVKZXCZUXKUYFXEXFZVUAXOXFZUXSVUJVCVUAYQVNVVOUXKUYFYMXFUXSUXMVUAUY
+        GYNUPZUYGVUAYNUPZUXJUXMVVRUSZUXQUXPUXJVUJUYIVVTVVNUYRUXKUYFUXLUVLYIXCZU
+        XSVUIVUFVVRVVSUSVVQVUGVUAUYGUVMYIYBYOZUXSUYIVCUYGYQVNUYSUYFUXLYMXFZUVNZ
+        UVPVUCUXSUWOUYGUXOVMUYHUXSVUAXGUNUYGXGUNUXOXGUNVUAUYGUWDUPZUXOUSUYGUXOV
+        MVNUXSVUAUXSVUAVVPUXSUXMVUAUYGVUEVVQVUGVWCVWAYOZUVQYPUXSUYGUXSUYGVWDUVR
+        YPVUDUXSVWEUXMUXOUXSUXMVWEUXSUXMVVRVWEVWAUXSVUAYRUNUYGYRUNVVRVWEUSVWFVW
+        BVUAUYGUVSYIYBUVTVUBYBVUAUYGUXOUWAYSUWBUXTUYAUXQUWCUYBUWPUWOUQJUXNUWEUW
+        FYSUXJVUNVUOVVBUWGYTUWSBUJCEGHIKLUWLOPQRSTUAUBUCUDAUWQUWNXJUWHYTADUWIUN
+        UWNUIVBVAUHBCDEHIUIOPQRSTUAUWJUWKYT $.
+    $}
+  $}
+
+  ${
+    $d A n $.  $d L n $.  $d Q n $.  $d S n $.  $d n ph $.
+    $d a b c d e f g h i j k l m n o p q r s t x y z $.
+    constrext2chn.q $e |- Q = ( CCfld |`s QQ ) $.
+    constrext2chn.l $e |- L = ( CCfld |`s S ) $.
+    constrext2chn.s $e |- S = ( CCfld fldGen ( QQ u. { A } ) ) $.
+    constrext2chn.a $e |- ( ph -> A e. Constr ) $.
+    $( If a constructible number is a member of some subfield ` L ` of ` CC ` ,
```

Comment:
Same comment as for `constrext2chnlem`. Also, I didn't check the proof, but it's surprising that it's so long given that `constrext2chn` looks exactly the same as `constrext2chnlem` but with an additional notation for `S`.